lib/results.js

```diff
@@ -157,8 +157,10 @@ function encodeResult (res, count) {
     if (res.at) {
         output += inner + 'at: ' + res.at + '\n';
     }
-    if (res.operator === 'error' && res.actual && res.actual.stack) {
-        var lines = String(res.actual.stack).split('\n');
+    var error = (res.actual instanceof Error ? res.actual : res.error);
```

Comment:
I'd prefer not to use `instanceof`, since it's not reliable across realms. (same as https://github.com/substack/tape/pull/330#discussion_r83794224)